.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "source-hugo/themes/techdocshugotheme"]
+	path = source-hugo/themes/techdocshugotheme
+	url = https://github.com/bholloway/techdocshugotheme.git
```

Comment:
How about a git url instead of HTTP

    url = git://github.com/bholloway/techdocshugotheme.git